### DISCUSSION TITLE
Control-u & Control-d doesn't work as it works in the console

### DISCUSSION THREAD

**Discussion by govardha** [vscode-neovim/vscode-neovim#1462] [Q&A]
Hello,

I am new to nvim, but I have a basic question.  When I use control-u/control-d to navigate up/down by half a page, and I get to the first portion of the file, in the linux console I go to the very first line of the file.  But in the neovim module I get stuck in the last half of the page.  I can then type in 1 to get to the first line.  

What am I missing?  

Thanks for any pointers.

**Comment by theol0403** [comment]
Problems currently with viewport scrolling. https://github.com/vscode-neovim/vscode-neovim/pull/1335
No good solution yet.
